lib/hcc-config.cmake.in

```diff
@@ -22,6 +24,16 @@ set_target_properties(hsa-runtime64 PROPERTIES
   INTERFACE_SYSTEM_INCLUDE_DIRECTORIES "${HSA_HEADER}"
 )
 
+pkg_check_modules(LIBUNWIND REQUIRED libunwind)
+add_library(${LIBUNDWIND_LIBRARIES} INTERFACE IMPORTED)
```

Comment:
This should be `add_library(unwind INTERFACE IMPORTED)`. Also the imported target needs to be created during build as well.